lib/webpacker_uploader/instance.rb

```diff
@@ -3,26 +3,37 @@
 require "mime-types"
 
 class WebpackerUploader::Instance
-  # TODO make this configurable
-  IGNORE_EXTENSION = %w[.map].freeze
-  public_constant :IGNORE_EXTENSION
+  DEFAULT_IGNORE_EXTENSION = %w[.map].freeze
 
   cattr_accessor(:logger) { ActiveSupport::TaggedLogging.new(ActiveSupport::Logger.new(STDOUT)) }
+  cattr_accessor(:ignored_extensions) { %w[.map] }
 
   def manifest
     @manifest ||= WebpackerUploader::Manifest.new
   end
 
   def upload!(provider)
+    extensions_to_ignore = if ignored_extensions.nil? || !ignored_extensions.is_a?(Array)
+                             logger.warning('Ignored extensions should be specified as an array.')
+                             logger.warning("Ignoring #{DEFAULT_IGNORE_EXTENSION} instead")
+
+                             DEFAULT_IGNORE_EXTENSION
+                           else
+                             ignored_extensions
+                           end
+
     manifest.assets.each do |name, path|
       path = path[1..-1]
       file_path = Rails.root.join("public", path)
 
-      if name.end_with?(*IGNORE_EXTENSION)
+      if name.end_with?(*extensions_to_ignore)
         logger.info("Skipping: #{file_path}")
       else
         logger.info("Processing: #{file_path}")
-        provider.upload!(path, file_path, content_type_for(path)) unless name.end_with?(*IGNORE_EXTENSION)
+
+        unless name.end_with?(*extensions_to_ignore)
+          provider.upload!(path, file_path, content_type_for(path))
+        end
```

Comment:
We can remove this if I think, it is covered by the outer clause.

Something like this will suffice:

```ruby
      if name.end_with?(*ignored_extensions)
        logger.info("Skipping: #{file_path}")
      else
        logger.info("Processing: #{file_path}")

        provider.upload!(path, file_path, content_type_for(path))
      end
```